src/packages/utils.ts

```diff
@@ -156,6 +164,33 @@ export const exportToSvg = async ({
   );
 };
 
+export const exportToClipboard = async (
+  opts: ExportOpts & {
+    mimeType?: string;
+    quality?: number;
+    type: "png" | "svg" | "text";
```

Comment:
I'd call it `json` instead of `text`. Text we may reserve for `copyAsText` action?

Reply:
Are you referring to `copy as text` option which shows up in context menu for copying on text elements?

Reply:
yep. But `json` also makes sense in general, since that's how we refer to this type of export everywhere.

Reply:
done